coalib/collecting/Collectors.py

```diff
@@ -170,7 +170,7 @@ def collect_bears(bear_dirs, bear_globs, kinds, log_printer,
 
     if warn_if_unused_glob:
```

Comment:
shortlog is not giving enough info. should be:

Collectors: Change "no bears" error message

also, does this PR fix no open issue? if it does, please link it in the commit message.